lib/proxy/helpers.rb

```diff
@@ -107,4 +107,19 @@ def remote_fqdn(forward_verify=true)
     end
   end
 
+  def ssl_client_cert(request)
+    if request.env.key?('SSL_CLIENT_CERT')
```

Comment:
IMHO this should look at the server in use and only test for that specific variable.

Reply:
I don't see any security threat, this way it can be a simple helper.

Reply:
When using Puma and providing a header named `SSL_CLIENT_CERT`, can you guarantee it won't end up in the env? I'd rather be safe than sorry and only check for what is known to be expected.